templates/blog/index.html.twig

```diff
@@ -5,6 +5,7 @@
 {% block meta_description "Découvrez nos articles techniques (ou non), rédigés par les membres de l'équipe d'Elao !" %}
 
 {% block content %}
+    <div class="container">
```

Comment:
I guess we miss the same container div for the article show template: 

https://elao.github.io/elao_/pr/439/blog/elao/sabrina-alternante-en-integration/

Reply:
I'm on it ☄️